jingo/__init__.py

```diff
@@ -3,15 +3,42 @@
 from __future__ import unicode_literals
 
 import functools
-import imp
 import logging
 import re
 
+from django.apps import apps
 from django.conf import settings
 from django.template.base import Origin, TemplateDoesNotExist
 from django.template.loader import BaseLoader
 from django.utils.importlib import import_module
 
+try:
+    import importlib.util
+    if hasattr(importlib.util, 'find_spec'):  # Py3>=3.4
+        def has_helpers(path):
+            return importlib.util.find_spec('helpers', path) is not None
+    else:  # Py3<3.4
+        def has_helpers(path):
+            # For Python 3.3, just try to import the module. Unfortunately,
+            # this changes the contract slightly for Python 3.3: if there is an
+            # module but this raises a legitimate ImportError, jingo will act
```

Comment:
I don't like this, but I was unable to find a way to replicate the behavior of `importlib.util.find_spec` (from >=3.4) or `imp.find_module('name', 'dotted.path')` (from 2.x) to tell me that a module was _present_, not whether it was, in fact, importable. At least without looking directly at the filesystem, which I'd rather not do because it's fragile and breaks not only unsupported-but-in-the-wild deployments (e.g. only ship `.pyc` files) but possibly supported ones (e.g. would a pexe work?).